app.rb

```diff
@@ -0,0 +1,30 @@
+require 'sinatra/base'
+require 'pg'
+
+class Chitter < Sinatra::Base 
+  get '/' do
+    erb :index
+  end
+
+  get '/sign-up' do
+    erb :sign_up
+  end
+
+  post 'add-database' do
```

Comment:
Sinatra didn't know the ditty. Adding a slash to both post routes seemed to work.
`post '/add-database' do`
`post '/new-peep' do`